LocationList.py

```diff
@@ -714,17 +1626,116 @@ def shop_address(shop_id, shelf_id):
+    ("Spirit Temple Big Mirror Flying Pot 4",                        ("Collectable",  0x06,  0x3C, None,                        'Recovery Heart',                        ("Spirit Temple", "Vanilla", "FlyingPot"))),
+    ("Spirit Temple Big Mirror Flying Pot 5",                        ("Collectable",  0x06,  0x3D, None,                        'Recovery Heart',                        ("Spirit Temple", "Vanilla", "FlyingPot"))),
+    ("Spirit Temple Big Mirror Flying Pot 6",                        ("Collectable",  0x06,  0x3E, None,                        'Recovery Heart',                        ("Spirit Temple", "Vanilla", "FlyingPot"))),
+    
+    # Spirit Temple MQ Pots
+    ("Spirit Temple MQ Lobby Pot 1",                                 ("Collectable",  0x06,  0x31, None,                        'Bombs (5)',                             ("Spirit Temple", "Master Quest", "Pot"))),
+    ("Spirit Temple MQ Lobby Pot 2",                                 ("Collectable",  0x06,  0x21, None,                        'Recovery Heart',                        ("Spirit Temple", "Master Quest", "Pot"))),
+    ("Spirit Temple MQ Lobby Pot 3",                                 ("Collectable",  0x06,  0x22, None,                        'Rupees (5)',                            ("Spirit Temple", "Master Quest", "Pot"))),
+    ("Spirit Temple MQ Lobby Pot 4",                                 ("Collectable",  0x06,  0x32, None,                        'Recovery Heart',                        ("Spirit Temple", "Master Quest", "Pot"))),
+    ("Spirit Temple MQ Child Torch Slugs Room Pot",                  ("Collectable",  0x06,  0x3F, ([0x2b123c4], None),         'Bombs (5)',                             ("Spirit Temple", "Master Quest", "Pot"))),
+    ("Spirit Temple MQ Child 3 Gibdo Room Pot 1",                    ("Collectable",  0x06,  0x33, None,                        'Rupees (5)',                            ("Spirit Temple", "Master Quest", "Pot"))),
+    ("Spirit Temple MQ Child 3 Gibdo Room Pot 2",                    ("Collectable",  0x06,  0x34, None,                        'Recovery Heart',                        ("Spirit Temple", "Master Quest", "Pot"))),
+    ("Spirit Temple MQ Child Stalfos Fight Pot 1",                   ("Collectable",  0x06,  0x30, None,                        'Rupees (5)',                            ("Spirit Temple", "Master Quest", "Pot"))),
+    ("Spirit Temple MQ Child Stalfos Fight Pot 2",                   ("Collectable",  0x06,  0x2F, None,                        'Recovery Heart',                        ("Spirit Temple", "Master Quest", "Pot"))),
+    ("Spirit Temple MQ Child Stalfos Fight Pot 3",                   ("Collectable",  0x06,  0x2E, None,                        'Recovery Heart',                        ("Spirit Temple", "Master Quest", "Pot"))),
+    #("Spirit Temple MQ Child Stalfos Fight Pot 4", ("Collectable", 0x06, 0x2D, None, 'N/A', ("Spirit Temple", "Master Quest", "Pot"))),
```

Comment:
Are these the same case as the guard house pots, where they have shared flags so can't be shuffled separately?

Reply:
No these 4 pots use separate flags. The flag is actually the "default" field in LocationList. In this case, 0x30, 0x2F, 0x2E. Pot 4 isn't shuffled because it's a fairy. I left a lot of these lying around like that in case one day I decided to go back and shuffle the fairies.

Reply:
You'll also find some places in LocationList where the "Address" field contains a Tuple and a list (as opposed to "None"). In this case, these are actors that did use a shared flag with something else in the scene, but it seemed reasonable and less confusing to simply replace it with an unused flag. Ex. Spirit Temple Before Child Climb Small Wooden Crate 1. In this case the address field contains a tuple. The first element of which is a list of ROM addresses corresponding to the Actor within the scene file, which will be patched with the flag set in the "default" field. The second field of the tuple is just "None"